src/main/java/com/internship/socialnetwork/model/UserFriendId.java

```diff
@@ -0,0 +1,18 @@
+package com.internship.socialnetwork.model;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+import java.io.Serializable;
+
+@Data
+@NoArgsConstructor
+@EqualsAndHashCode
+public class UserFriendId implements Serializable {
```

Comment:
Here you should add `@Embeddable` so it can be used in other class as embeded id